### DISCUSSION TITLE
提示已经安装了，但是实际没有防撤回的

### DISCUSSION THREAD

**Discussion by blacnk-oss** [huiyadanli/RevokeMsgPatcher#409] [Q&A]
![image](https://user-images.githubusercontent.com/112036834/186543977-d6df1e31-3336-4929-8b0c-807fece23da4.png)
重装了微信还是不行，微信版本是3.7.6.29

**Comment by huiyadanli** [comment]
是否获取了最新的补丁信息？

**Reply by lckb123** [reply]
今天微信刚更新，看不到自己撤回的消息，但是能看见别人的。其实自己的看不i看见无所谓。哈哈哈
